src/routes/forms.test.js

```diff
@@ -427,7 +425,9 @@ describe('Forms route', () => {
     )
 
     test('Testing GET /forms/{id} route with an id that is not found returns 404 Not found', async () => {
-      jest.mocked(getForm).mockResolvedValue(undefined)
+      jest
+        .mocked(getForm)
+        .mockRejectedValue(Boom.notFound(`Form with slug '${id}' not found`))
```

Comment:
Oh no, this message is for slugs but we're testing IDs here

It's one of the side effects I mentioned in https://github.com/DEFRA/forms-manager/pull/148#discussion_r1617261250

By mocking `getForm()` we're skipping coverage for the actual error message lines